packages/mdc-chips/chip/adapter.js

```diff
@@ -47,6 +47,12 @@ class MDCChipAdapter {
    */
   hasClass(className) {}
 
+  /**
+   * Returns true if contains the checkmark class.
+   * @return {boolean}
+   */
+  hasCheckmark() {}
```

Comment:
Just realized...if we're restricting the opacity changes to filter chips in CSS, then we don't need to check for checkmarks in the JS anymore :D

Reply:
Done!